apps/chat/src/store/conversations/conversations.selectors.ts

```diff
@@ -147,8 +147,11 @@ export const selectFilteredFolders = createSelector(
 export const selectLastConversation = createSelector(
   [selectConversations],
   (conversations): ConversationInfo | undefined => {
-    if (!conversations.length) return undefined;
-    return sortByDateAndName([...conversations])[0];
+    const ownConversations = conversations.filter(
+      (conv) => !isEntityExternal(conv),
```

Comment:
Do we want to select entities inside external folders? Maybe it's better to filter them with `!isEntityOrParentExternal` ?